src/modules/launcher/Wox.Plugin/loc/cs/src/modules/launcher/Wox.Plugin/Properties/Resources.resx.lcl

```diff
@@ -34,7 +34,7 @@
         </Str>
         <Disp Icon="Str" />
         <Cmts>
-          <Cmt Name="Dev"><![CDATA[paramaters: fromPath, toPath]]></Cmt>
+          <Cmt Name="Dev"><![CDATA[parameters: fromPath, toPath]]></Cmt>
```

Comment:
@arjunbalgovind the LCL parameter misspelling, is this something we have to worry about?

Reply:
It seems to be a reference comment for the translators for understanding what {0} and {1} is. So technically correcting the spelling shouldn't have any issues, but I'd still be wary of making changes here. Might be something to double check with Bruno.